eng/Directory.Build.Common.props

```diff
@@ -177,13 +180,52 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
+  <!-- Disable documentation for projects that need fixes before this can be enabled. -->
+  <PropertyGroup Condition="
```

Comment:
Is the goal to file an issue and fix these or do we plan to just forget about these? If we plan to forget about them I think it would be better to set the property directly in their project instead of the common props so we don't add this noise to the common props as we start to phase out track 1.

Reply:
The goal is to fix these errors. Not getting reference documentation has been a complaint from users.
Having this info in one place allows for a quick view of all the packages that are broken.

Reply:
OK if we plan to fix these for track 1 then I'm OK having them in one place like this. Please file an issue and link to that issue here in a comment.

Reply:
I am opening bugs like this one - https://github.com/Azure/azure-sdk-for-net/issues/23882